pyvista/plotting/plotting.py

```diff
@@ -3809,9 +3809,19 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
 
     @wraps(ScalarBars.remove_scalar_bar)
     def remove_scalar_bar(self, *args, **kwargs):
-        """Remove the active scalar bar."""
+        """Remove the active scalar bar.
```

Comment:
docstring needs to be before the python code.
```suggestion
        """Remove the active scalar bar.
        
        Examples
        --------
        
        Remove scalar bar from a plotter.
        >>> p = pv.Plotter()
        >>> p.add_mesh(mesh, cmap='coolwarm')
        >>> p.remove_scalar_bar()
        >>> p.show()
        """
```

Reply:
This also needs to be moved to `ScalarBars`. I'm moving now...